@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -852,6 +846,7 @@ watch(
       vifs: getExistingVifs(template),
       selectedVdi: undefined,
       installMode: undefined,
+      bootFirmware: template.boot.firmware !== undefined ? template.boot.firmware : 'bios',
```

Comment:
Can it be `null`? Otherwise:

```suggestion
      bootFirmware: template.boot.firmware ?? 'bios',
```